### DISCUSSION TITLE
Resumability

### DISCUSSION THREAD

**Discussion by saolof** [leptos-rs/leptos#587] [Ideas]
The topic of Qwik-like resumability came up during the stream with Ryan Carniato a while ago, since signals index into an arena which technically means that in the current implementation (ignoring mid-execution stack state) it basically gets a serialization of the current underlying state for free.

I understand that this is probably not a primary design goal for Leptos, but I thought it would be fun to create a discussion topic for it

**Comment by gbj** [comment]
See also [this issue](https://github.com/leptos-rs/leptos/issues/12).
